src/App.js

```diff
@@ -1,12 +1,29 @@
 /* eslint-disable react/prefer-stateless-function */
 import './App.css';
 import React from 'react';
-import Calculator from './components/Calculator';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import Header from './components/Header';
+import QuotePage from './pages/QuotePage';
+import CalculatorPage from './pages/CalculatorPage';
+import HomePage from './pages/HomePage';
 
 export default function App() {
   return (
     <div className="app-container">
-      <Calculator />
+      <Router>
+        <Header />
+        <div className="body-container">
+          <Switch>
+            <Route path="/quote" component={QuotePage} />
+            <Route path="/calculator" component={CalculatorPage} />
+            <Route path="/home" component={HomePage} />
```

Comment:
- [ ] Kindly update your default route to the `Home` page. Currently, when visiting the `/` route a blank white screen appears instead of the `Home` page.